### PR TITLE
Prepare release

### PR DESCRIPTION
**Pull request by bfmatei**
- Update README files
- Remove Angular package
- Update global version
- Enable inline sources in sourcemaps
- Update dependencies
- Added keywords to packages
- Added missing export

### CODE REVIEW

File: lerna.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "packages": ["packages/*", "demo"],
-  "version": "1.0.0-beta2",
+  "version": "1.0.0-beta3",
```
Comment (domasx2) [inline lerna.json:3]: I think lerna does this atuomatically as a separate commit

File: packages/core/README.md [hunk 2]
```diff
@@ -134,16 +164,40 @@ The `api` property on the Faro instance contains all the necessary methods to pu
...
-    {
-      span: mySpan,
-    }
+    { skipDedupe: true }
   );
   ```
 
+## Traces
+
+- `pushTrace` - is a method for manually sending traces through the transports. The method only accepts a mandatory
```
Comment (domasx2) [inline packages/core/README.md:173]:
```suggestion
- `pushTraces` - is a method for manually sending traces through the transports. The method only accepts a mandatory
```

File: packages/core/README.md [hunk 3]
```diff
@@ -172,9 +226,9 @@ initializeFaro({
 
 Metas are objects that will be attached to every event that is triggered by the API.
 
-Out of the box, only one meta is provided: `sdk` which contains information about the Faro instance and its version. Additional
-metas can be provided by external packages like [@grafana/agent-meta-browser](https://github.com/grafana/faro-web-sdk/tree/main/packages/meta-browser)
-and [@grafana/agent-meta-page](https://github.com/grafana/faro-web-sdk/tree/main/packages/meta-page).
+Out of the box, only one meta is provided: `sdk` which contains information about the Faro instance and its version.
+
+Additional metas should be provided by platform packages like [@grafana/faro-web-sdk](https://github.com/grafana/faro-web-sdk/tree/main/).
```
Comment (domasx2) [inline packages/core/README.md:231]:
```suggestion
Additional metas may be provided by platform packages like [@grafana/faro-web-sdk](https://github.com/grafana/faro-web-sdk/tree/main/).
```

File: lerna.json (hunk 1 above)
Comment (domasx2) [reply lerna.json:3]: I mean, when you run `npx lerna version`

**Review by domasx2 — APPROVED** [review]
Minor typo. Also should not manually bump version in `lerna.json`, but use `npx lerna version`

File: lerna.json (hunk 1 above)
Comment (bfmatei) [reply lerna.json]: Reverted

**Pull request merged**